django/chat/templates/chat/components/options_2_summarize.html

```diff
@@ -19,11 +19,21 @@
   </div>
   <div class="px-3">{{ options_form.summarize_language }}</div>
 
+  <div class="mx-3 mt-2 form-switch form-check">
+    {{ options_form.summarize_gender_neutral }}
+    <label class="form-check-label small ms-1"
+           for="{{ options_form.summarize_gender_neutral.auto_id }}">{% trans "Avoid gendered pronouns" %}</label>
+  </div>
+
+  <label for="{{ options_form.summarize_instructions.auto_id }}"
+         class="form-label px-3 mt-2 mb-1 small">{% trans "Additional instructions" %}</label>
+  <div class="px-3">{{ options_form.summarize_instructions }}</div>
+
   <label for="{{ options_form.summarize_prompt.auto_id }}"
          class="form-label px-3 mt-2 mb-1 small">
     {% trans "Custom summarize prompt" %}
     <br>
-    {% trans "(overrides length/language)" %}
+    {% trans "(overrides all previous settings)" %}
```

Comment:
Is this still necessary, if it overrides "additional instructions"? What would be an example of behaviour that wouldn't be well-encoded in "Additional Instructions", but would be better enforced with an override prompt here?

Reply:
After taking a closer look through the summarize_long_text code, I better understand the function of the custom/override prompt, but I still wonder if it's presented in a way where the benefit is obvious to users. Should we include an example of a summarization prompt as a placeholder or something like that?

Reply:
Let's improve the UX later by having a toggle between "basic/advanced" or something.